### ISSUE TITLE
[Feature Request] Haptic API/OSC/Support

### ISSUE THREAD

**Issue by Gatorzgaming** [SlimeVR/SlimeVR-Server#946]
Haptics are becoming a growing element of VR, having an API for them built into slimeVR server has a lot of potential and opens the door up for more slimeVR integration. I am unsure if this is considered out of scope, but I wanted to make sure there was at least an issue for it. As I have with SlimeVR wrangler haptic support. https://github.com/carl-anders/slimevr-wrangler/issues/39

**Comment by DatanIMU** [comment]
Yes, there is a Chinese company called Datan that has implemented locomotion like functions through IMU trackers. 
I'm wondered if SlimeVR can do this.

Here is there implementation.
https://www.bilibili.com/video/BV1J2421K7ce/?spm_id_from=333.999.0.0&vd_source=4fcdd54f8c9cb06eb78b16cf4e3855cd

Pls give reply, very appreciate.

**Comment by digitalf0x** [comment]
Since this was closed as "completed", are there any commits or other documentation on this?  My initial searching didn't reveal anything.

I have several friends either using SlimeVR or looking to get it, and I'd be happy to try to make Haptic Pancake work for SlimeVR too, not just Tundra and Vive Trackers.

**Comment by loucass003** [comment]
its closed as not planned atm. our team dont have the capacity for it

**Comment by digitalf0x** [comment]
Aww, though that's understandable.  If this changes in the future, feel free to ping me and I'll see how I can help (I'm not currently equipped to write the entire support myself).

Meanwhile, good luck with everything!
